src/utils/client.util.ts

```diff
@@ -0,0 +1,11 @@
+import axios from "axios";
+
+// axios client
+const apiClient = axios.create();
```

Comment:
Can you remove comment I think there is no need for this

Reply:
- Done